rpc/wordvault/api.proto

```diff
@@ -107,6 +107,21 @@ message NextScheduledCountRequest {
 
 message NextScheduledBreakdown { map<string, uint32> breakdown = 1; }
 
+message NextScheduledCountByDeckRequest {
+  bool only_overdue = 1;
+  string timezone = 2;
+  string lexicon = 3;
+}
+
+message DeckBreakdown {
+  uint64 deck_id = 1;
+  map<string, uint32> breakdown = 2;
+}
+
+message NextScheduledCountByDeckResponse {
+  repeated DeckBreakdown breakdowns = 1;
```

Comment:
@BenMusch Is the intention to eventually merge both of these endpoints into one? I think it would be good to have one endpoint that just breaks it down by deck eventually. (The one you just built).

Reply:
Yep that's the plan, just keeping both since we need to hook up the UI to the new one first